interfaces/apparmor/backend.go

```diff
@@ -259,6 +259,21 @@ func setupSnapConfineReexec(coreInfo *snap.Info) error {
 	return nil
 }
 
+// nsProfile returns name of the apparmor profile for snap-update-ns for a given snap.
+func nsProfile(snapName string) string {
+	return fmt.Sprintf("snap-update-ns.%s", snapName)
+}
+
+// profileGlobs returns a list of globs that describe the apparmor profiles of
+// a given snap.
+//
+// Currently the list is just a pair. The first glob describes profiles for all
+// apps and hooks while the second profile describes the snap-update-ns profile
+// for the whole snap.
+func profileGlobs(snapName string) []string {
+	return []string{interfaces.SecurityTagGlob(snapName), nsProfile(snapName)}
```

Comment:
The first glob is now `snap.<name>.*`, previously it was `snap*.<name>*` (no dot after `<name>`). Leaving the dot aside, any idea why the old glob started with `snap*.<name>`? By the looks of it (and the lack of dot), `snap-update-ns.<name>` would be picked up by that glob too.

Reply:
Ha, interesting observation. No I don't know. Perhaps because it was before we had support for multiple globs and we had to capture one of the hooks?